iree/samples/static_library/create_c_module.c

```diff
@@ -0,0 +1,12 @@
+// Copyright 2021 The IREE Authors
+//
+// Licensed under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include "iree/samples/static_library_emitc/simple_mul_emitc.h"
```

Comment:
```suggestion
#include "iree/samples/static_library/simple_mul_emitc.h"
```

We missed do adjust this include after we merged the two static library samples.